client/src/components/Tombstones.js

```diff
@@ -17,11 +16,14 @@ const UnlabeledTombstone = ({ items }) => (
 );
 
 const LabeledTombstone = ({ labels_and_items }) => (
-  <dl className="dl-horizontal tombstone-data-list">
+  <dl className="row tombstone-data-list">
     {_.map(labels_and_items, ([label, item], ix) => (
       <Fragment key={ix}>
-        <dt>{label}</dt>
-        <dd>{item}</dd>
+        <dt className="col-2">{label}</dt>
+        <dd className="col-10">{item}</dd>
```

Comment:
Not handling small screens. Probably want these to be something like `md`, and to give both the label and item 12's for `sm`. Should probably be switched to `frow`/`fcol`s too?
![image](https://user-images.githubusercontent.com/11301919/107702393-2a8a7080-6c88-11eb-94ca-1a539c58b2c6.png)

There was one other `dl-horizontal` in this PR I think, apply the same fixes there.